app/models/name.rb

```diff
@@ -350,25 +350,24 @@ class Name < AbstractModel
 
   # enum definitions for use by simple_enum gem
   # Do not change the integer associated with a value
-  enum rank:
-        {
-          Form: 1,
-          Variety: 2,
-          Subspecies: 3,
-          Species: 4,
-          Stirps: 5,
-          Subsection: 6,
-          Section: 7,
-          Subgenus: 8,
-          Genus: 9,
-          Family: 10,
-          Order: 11,
-          Class: 12,
-          Phylum: 13,
-          Kingdom: 14,
-          Domain: 15,
-          Group: 16 # used for both "group" and "clade"
-        }
+  enum :rank, {
```

Comment:
@nimmolo @mo-nathan @pellaea: Heads-up.
This change was automatic --without notifying me-- I guess via the combination of VSCode and RuboCop.
That's disturbing and probably reason for closer scrutiny of PRs from users of VSCode (@nimmolo, @JoeCohen)
But the change is okay: "Defining enums with keyword arguments syntax is deprecated and will be removed in Rails 8.0. Positional arguments should be used instead"
See https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsenumsyntax